### ISSUE TITLE
Characters in SMS not rendered properly

### ISSUE THREAD

**Issue by pregnantboy** [opengovsg/postmangovsg#522]
**Describe the bug**
Square brackets in SMS not rendered properly

**To Reproduce**
Send a message with []
Please also check if other characters are affected

**Screenshots**
![image](https://user-images.githubusercontent.com/10072985/87308469-dd0d2000-c54d-11ea-86aa-4424f6ea19bb.png)

**Comment by jeantanzj** [comment]
I created a campaign to verify the bug, but I could receive the message correctly. 

*iphone xr, hello [you] renders wrongly, [bed request] renders correctly*
![image0](https://user-images.githubusercontent.com/33819199/87342298-9800e200-c57d-11ea-9037-1adf489ff1d8.png)

*pixel3, both render correctly*
![Screenshot_20200713-220049](https://user-images.githubusercontent.com/33819199/87342309-99caa580-c57d-11ea-886c-58f5fbfde9c9.png)

*ione plus 7t, both  hello [you] and [bed request] render wrongly*
![Screenshot_20200713-215738](https://user-images.githubusercontent.com/33819199/87342373-b6ff7400-c57d-11ea-9e85-6617fccbbd4e.jpg)


After a while, I resent that same campaign to myself, and my phone rendered it with the incorrect characters.

- Smart Encoding is switched on in Twilio
- Twilio logs show that the messages are GSM encoded (not UCS2)

I raised a support ticket with Twilio, lets see if they'll have any insights.
